api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/OpcNodeOnEndpointApiModel.cs

```diff
@@ -0,0 +1,55 @@
+﻿// ------------------------------------------------------------
```

Comment:
I think we can reuse PublishedNodeApiModel here

Reply:
yes